src/actions/effect/leveled/loop.ts

```diff
@@ -1,6 +1,10 @@
 import LeveledEffectAction from "../EffectActions/LeveledEffectAction";
 
-
+/**
+ * @description Delivers a video or animated GIF that contains additional loops of the video/GIF.
+ * @class LoopEffectAction
+ * @augments LeveledEffectAction
```

Comment:
we should have the same convention of @description at the end

Reply:
As discussed